versioned_docs/version-2.x/advanced/analytics/getting-started.mdx

```diff
@@ -94,6 +94,30 @@ module.exports = {
 };
 ```
 
+### Retrieve authorized cookie services in analytics
+
+Currently in Front-Commerce, in order to send authorized cookie services to
+analytics services, those services also need to be defined in
```

Comment:
**nitpick:** Its not very clear, as it feel that they're "automatically sent" to analytics services. They're not. So far, only GTM supports this if I'm not mistaken.

**suggestion:** mention where it is sent in Analytics plugins

```suggestion
In Front-Commerce, to allow authorized cookie services to
be injected in analytics services `settings(authorization, otherAuthorizations)` callback (as its 2nd parameter), those services also need to be defined in
```